tgstation.dme

```diff
@@ -14,7 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
-#include "_maps\tgstation2.dm"
+#include "_maps\runtimestation.dm"

```

Comment:
A noob who follows the download/install/launch procedure might not expect to be placed into the debug map. At the very least the tutorial in the readme and on the wiki should be updated with a sentence on switching from runtime to tgstation or another true map.